import-service/serverless.ts

```diff
@@ -40,8 +52,52 @@ const serverlessConfiguration: Serverless = {
         Action: ['s3:*'],
         Resource: [`arn:aws:s3:::${BUCKET}/*`],
       },
+      {
+        Effect: 'Allow',
+        Action: ['sqs:*'],
+        Resource: [
+          {
+            'Fn::GetAtt': ['SQSQueue', 'Arn'],
+          },
+        ],
+      },
+      {
+        Effect: 'Allow',
+        Action: ['sns:*'],
+        Resource: [
+          {
+            Ref: 'SNSTopic'
+          }
+        ],
+      },
     ],
   },
+  resources: {
+    Resources: {
```

Comment:
From my point of view it's better to create all these stuff in product-service. I mean SQS, SNS and catalogBatchProcess. Because SQS it's a queue of products. And in the future any other service can send messages in this queue, not only import service. But you can do whatever you want. It's just my suggestions.